ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/StreamedQueryResource.java

```diff
@@ -353,6 +354,14 @@ private EndpointResponse handlePushQuery(
     final ConfiguredStatement<Query> configured = ConfiguredStatement
         .of(statement, SessionConfig.of(ksqlConfig, streamsProperties));
 
+    final int numPushQueries = QueryCapacityUtil.getNumLivePushQueries(ksqlEngine);
```

Comment:
This is just one of the endpoints `/query`.  We want both the others.  There's already a check for `/query-stream`, but it's inconsistent with this.  I'm referring to the code in `io.confluent.ksql.api.server.Server.registerQuery`.  It seems like it was going to be part of some greater organization of connections, but wasn't extended to other endpoints.  

@vcrfxia and @agavra  I think reviewed some of that original code and can maybe comment.  If we want to keep that, then we can declare `/query-stream` done. Otherwise removing it and being consistent with this code would be good.

We should at least be sure to add this to `/ws/query` as well.

Reply:
I have a meta-question here, why do we track this at the endpoint instead of in the engine? Seems a little weird to me...